cli/src/config/templates.toml

```diff
@@ -97,6 +97,13 @@ if(
 '''
 
 [template-aliases]
+'hyperlink(url, text)' = '''
```

Comment:
Is `text` or `display` better? I'm fine with either. I'm mentioning it only because I've seen it called `display` in someone else's alias and I want to make sure we consider that name. I think I'm leaning towards `text`.